app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt

```diff
@@ -24,8 +23,8 @@ import cm.aptoide.pt.aptoide_ui.animations.animatedComposable
 import cm.aptoide.pt.extensions.PreviewDark
```

Comment:
rebase and change to the animatedComposable function on the extensions module

Reply:
This change was in an old commit from an already merged PR